### PR TITLE
✨Source Facebook Marketing: Add support for ad rule libraries

### PR DESCRIPTION
**Pull request by nicolasgpvoodoo**
## What
Linked issue: #32640
This PR adds support for the retrieval of [ad rule](https://developers.facebook.com/docs/marketing-api/reference/ad-rule/) libraries.

## How
Usage of the standard `get_ad_rule_libraries` function [from the SDK](https://github.com/facebook/facebook-python-business-sdk/blob/be38bc64b76e3f292e88d60fa4f5403e24ede8f9/facebook_business/adobjects/adaccount.py#L1029), and addition of a new dedicated stream.

## Recommended reading order
1. `airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/streams.py`
2. `airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json`
3. `airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py`
4. `airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/__init__.py`

## 🚨 User Impact 🚨
Addition of a new stream. No breaking changes.
Minor version bump (new feature).

### CODE REVIEW

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "properties" : {
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json:2]: Please make sure your schema follow https://docs.airbyte.com/connector-development/schema-reference

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Can you update the metadata file and the doc changelog entry? Thanks

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json (hunk 1 above)
Comment (nicolasgpvoodoo) [reply airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json:5]: Done, sorry about that.

**Comment by nicolasgpvoodoo** [comment]
> Can you update the metadata file and the doc changelog entry? Thanks

Done

**Review by lazebnyi — CHANGES_REQUESTED** [review]
@nicolasgpvoodoo hey 
Thanks for your PR 
Can you provide access to contribute to your PR (Checkbox under reviewers/assignees section)

**Comment by nicolasgpvoodoo** [comment]
> @nicolasgpvoodoo hey Thanks for your PR Can you provide access to contribute to your PR (Checkbox under reviewers/assignees section)

Unfortunately I don't see the checkbox. I think it's due to my company's security policy. I did a rebase of the PR though. If that's not enough, I can re-open this PR from a public fork, there I should have the checkbox available.

**Comment by lazebnyi** [comment]
@nicolasgpvoodoo Yes, let's re-open it with public fork. I should able to add commits to your branch.

**Review by lazebnyi — CHANGES_REQUESTED** [review]
The PR should be reopened with a public fork to enable running CI tests.

**Comment by marcosmarxm** [comment]
Closing as the origin branch was deleted.

**Pull request closed without merging**